.editorconfig

```diff
@@ -0,0 +1,250 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+# top-most EditorConfig file
+root = true
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+# Generated code
```

Comment:
Why do we need any suggestions at all for generated code? Any changes would have to be in the generator.

Reply:
We've still got a lot of entries in this section - do we need them at all? (Perhaps you just haven't pushed your change?)

Reply:
Removed the not required entries for generated code.